### ISSUE TITLE
"..." button doesn't work in badge dropdown

### ISSUE THREAD

**Issue by bviktor** [akiko-pusu/redmine_issue_badge#136]
There used to be an "Issues assigned to me" (or something like that) link in the badge dropdown when there were more issues than the max issues displayed. Now there's only a "..." text, but there's no link to click on anymore so we're unable to show all the issues, unless manually performing an issue query.

```
Environment:
  Redmine version                4.1.1.stable
  Ruby version                   2.6.5-p114 (2019-10-01) [x86_64-linux]
  Rails version                  5.2.4.2
  Environment                    production
  Database adapter               PostgreSQL
  Mailer queue                   ActiveJob::QueueAdapters::AsyncAdapter
  Mailer delivery                smtp
SCM:
  Git                            1.8.3.1
  Filesystem                     
Redmine plugins:
  computed_custom_field          1.0.7
  redmine_agile                  1.5.3
  redmine_checklists             3.1.17
  redmine_contacts               4.3.0
  redmine_issue_badge            0.1.3
  redmine_latex_mathjax          0.3.0
  redmine_zenedit                1.0.3
  redmineup_tags                 2.0.8
  wiki_graphviz_plugin           0.8.0
```

**Comment by akiko-pusu** [comment]
Hi, thanks for the reporting issue. I'll confirm it.

**Comment by akiko-pusu** [comment]
I've just confirmed this bug and fixed on commit: https://github.com/akiko-pusu/redmine_issue_badge/pull/137
Soon I'm going to merge into master and release a new version.

Thanks for your reporting!

**Comment by akiko-pusu** [comment]
"..." is not a button and you can go to more issues by clicking the bottom icon named "assigned to me".

<img width="342" alt="fixed" src="https://user-images.githubusercontent.com/122621/86996398-87034a00-c1e6-11ea-8dcd-740604afaca0.png">

**Comment by bviktor** [comment]
Awesome, thanks a lot! Can't wait for the next release :)

**Comment by akiko-pusu** [comment]
Released v0.1.4.
https://github.com/akiko-pusu/redmine_issue_badge/releases/tag/0.1.4

Thanks a lot! Wish you the best!

**Comment by bviktor** [comment]
Awesome, just deployed on our prod instance, works fine. Thanks a lot for the quick resolution!
